build.sh

```diff
@@ -163,6 +163,14 @@ fi
 if [[ -z ${CCACHE} ]]; then
     CCACHE=ccache
 fi
+if [[ -z ${USE_AVX2KI} ]]; then
+    ## Disable it by default
+    USE_AVX2KI=OFF
+fi
+
+if [[ "${MACHINE_TYPE}" == "aarch64"  && -f ${STARROCKS_THIRDPARTY}/installed/ksl/lib/libavx2ki.so ]]; then
```

Comment:
we shall implement the behavior that if `USE_AVX2KI` is empty, then auto detect the ON/OFF based on the cpu arch and avx2ki.so file. Otherwise respect the USE_AVX2KI specified by the user and possibly force turning it off if related thirdparty library can't be found, and a warning message is printed.

Reply:
one more thing, I see when this option is ON on arm platform, the final be binary will be dynamically linked to the libavx2ki.so, so this .so should be in the final `output/be/lib` directory as well.